packages/web-runtime/src/store/user.js

```diff
@@ -263,7 +263,7 @@ const mutations = {
   SET_QUOTA(state, quota) {
     // Turn strings into ints
     quota.free = parseInt(quota.free)
-    quota.relative = parseInt(quota.relative)
+    quota.relative = parseFloat(quota.relative)
```

Comment:
@fschade took me a bit longer because I needed to investigate why the relative used storage (which the backend reports correctly) always ends up as `0` in the store 💩 also fixed the fixtures where this was reflected, too